### ISSUE TITLE
Crash on 1.2.0 on Linux

### ISSUE THREAD

**Issue by aeris** [gw2scratch/evtc#134]
Hello!
I upgrade from v1.1.1 to v1.2.0 and now I get a crash on start

```
$ ./ArcdpsLogManager    
Exception in Gtk# callback delegate
  Note: Applications can use GLib.ExceptionManager.UnhandledException to handle the exception.
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at GW2Scratch.ArcdpsLogManager.Logs.Updates.LogDataUpdater.<>c.<.cctor>b__3_12(LogData log)
   at GW2Scratch.ArcdpsLogManager.Logs.Updates.LogUpdate.FuncLogFilter.FilterLog(LogData log)
   at GW2Scratch.ArcdpsLogManager.Logs.Updates.LogDataUpdater.<>c__DisplayClass1_0.<GetUpdates>b__2(LogUpdate u)
   at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Func`2 predicate, Boolean& found)
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at GW2Scratch.ArcdpsLogManager.Logs.Updates.LogDataUpdater.<>c.<GetUpdates>b__1_1(LogData log)
   at System.Linq.Lookup`2.Create(IEnumerable`1 source, Func`2 keySelector, IEqualityComparer`1 comparer)
   at System.Linq.GroupedEnumerable`2.GetEnumerator()
   at GW2Scratch.ArcdpsLogManager.Logs.Updates.LogDataUpdater.GetUpdates(IEnumerable`1 logs)+MoveNext()
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source)
   at GW2Scratch.ArcdpsLogManager.ManagerForm.<>c__DisplayClass53_0.<ConstructMenuBar>b__1(Object sender, EventArgs args)
   at GW2Scratch.ArcdpsLogManager.ManagerForm.<ReloadLogs>b__55_2()
   at Eto.GtkSharp.Forms.ApplicationHandler.<>c__DisplayClass19_0.<Invoke>b__0(Object <p0>, EventArgs <p1>)
   at Gtk.Application.InvokeCB.Invoke()
   at GLib.Timeout.TimeoutProxy.Handler()
   at GLib.ExceptionManager.RaiseUnhandledException(Exception e, Boolean is_terminal)
   at GLib.Timeout.TimeoutProxy.Handler()
   at Gtk.Application.Run()
   at Eto.GtkSharp.Forms.ApplicationHandler.Run()
   at Eto.Forms.Application.Run(Form mainForm)
   at GW2Scratch.ArcdpsLogManager.Gtk.Program.Main()
   ```

Starting v1.1.1 is fine.

**Comment by Sejsel** [comment]
This is a problem that also affects the Windows version.

The problem is a missing null check [here](https://github.com/gw2scratch/evtc/blob/master/ArcdpsLogManager/Logs/Updates/LogDataUpdater.cs#L59) and [here](https://github.com/gw2scratch/evtc/blob/master/ArcdpsLogManager/Logs/Updates/LogDataUpdater.cs#L63).

I should be able to release a fixed version out on Thursday (sadly I don't have time until then), thanks for the report!

**Comment by Sejsel** [comment]
Should be fixed in 53935238f16f9e51f381704a04a7bda4a5b55fb9. I will try to get it released today.
